application/forms/Config/General/PasswordPolicyConfigForm.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Icinga\Forms\Config\General;
+
+use Icinga\Application\Hook;
+use Icinga\Web\Form;
+
+/**
+ * Configuration form for password policy selection
+ *
+ * This form is not used directly but as subform for the {@link GeneralConfigForm}.
+ */
+class PasswordPolicyConfigForm extends Form
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function init(): void
+    {
+        $this->setName('form_config_general_password_policy');
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return  $this
+     */
+    public function createElements(array $formData)
```

Comment:
```suggestion
    public function createElements(array $formData): self
```

(https://github.com/Icinga/developer-guidelines-drafts/issues/83)